content/altinn-studio/concepts/payment/_index.nb.md

```diff
@@ -0,0 +1,26 @@
+---
+title: Betaling
+description: Det er mulig å konfigurere brukerbetaling for tjenester som er gebyrbelagte eller der det er tillatt å kreve betaling.
+weight: 10
+---
+
+Betaling er et prosessteg som kan legges til applikasjonen. 
+Ved inngang til betalingssteget omdirigeres brukeren til en ekstern betalingsleverandør, og returneres til tjenesten når betalingen er fullført.
+
+Ordregrunnlag og detaljer om betalingen lagres i et dataelement som JSON (paymentInformation.json), og kan lastes ned via API etter insending av skjemaet. Det blir også generert en PDF-kvittering.
+Tjenesteleverandør må selv håndtere oppbevaring av betalingsinformasjon i henhold til bokføringsloven.
```

Comment:
Hva mener vi med "tjenesteleverandør" her? Refererer det til tjenesteeier? Hvis det er tilfelle, bør vi tydeliggjøre dette ved å skrive "Det er du som tjenesteeier", for å unngå forvirring rundt begrepene tjenesteleverandør/tjenesteeier. Det er viktig at vi er klare på hvem vi henvender oss til og bruker begreper og skrive direkte til målgruppen.

Eller har vi endret begrepet fra tjenesteeier til tjenesteleverandør som felles begrep?

Reply:
Glipp dette. Takk! Fikser.